Addition to vagrant (#42)
Simulation-Software-Engineering/Lecture-Material: virtualization-and-containers/material/exercise_vm_containers_text.md

- [Vagrant Homepage](https://www.vagrantup.com/)
- [Vagrant Introduction](https://www.vagrantup.com/intro)
- [VirtualBox Manual](https://www.virtualbox.org/manual/UserManual.html)

> - Issue with windows and vagrant: ```VBoxManage.exe: error: Details: code E_FAIL (0x80004005), component ConsoleWrap, interface IConsole```

Reviewer: ```suggestion
- If you are using Vagrant on Windows you might run into the following issue/error message: 

  ```powershell
  VBoxManage.exe: error: Details: code E_FAIL (0x80004005), component ConsoleWrap, interface IConsole
  ```

```

Maybe we could rephrase this part a bit? We should also wrap PowerShell commands in an fenced code block with PowerShell identifier.

>     - Solution described in this [stackoverflow thread](https://stackoverflow.com/questions/37955942/vagrant-up-vboxmanage-exe-error-vt-x-is-not-available-verr-vmx-no-vmx-code)

Reviewer: ```suggestion
    - The short version of the solution is given below. The detailed solution is described in this [Stack Overflow thread](https://stackoverflow.com/questions/37955942/vagrant-up-vboxmanage-exe-error-vt-x-is-not-available-verr-vmx-no-vmx-code).
```
Some rephrasing.

>     - Basically run (in powershell as admin): ```bcdedit /set hypervisorlaunchtype off```

Reviewer: ```suggestion
          1. As admin, run the following command in a PowerShell:
    
            ```powershell
            bcdedit /set hypervisorlaunchtype off
            ```
```
Again some rephrasing and fenced code block with PowerShell identifer.

>     - Restart computer

Reviewer: ```suggestion
        2. Restart your computer
```
Some rephrasing.

>     - And run (in powershell as admin): ```bcdedit /set hypervisorlaunchtype auto```

Reviewer: ```suggestion
        3. After the restart, run the following command as admin a PowerShell:
        
           ```powershell 
           bcdedit /set hypervisorlaunchtype auto
           ```

```
Again some rephrasing and fenced code block with PowerShell identifer.

- [Vagrant Homepage](https://www.vagrantup.com/)
- [Vagrant Introduction](https://www.vagrantup.com/intro)
- [VirtualBox Manual](https://www.virtualbox.org/manual/UserManual.html)
- If you are using Vagrant on Windows you might run into the following issue/error message:

  ```powershell

>   VBoxManage.exe: error: Details: code E_FAIL (0x80004005), component ConsoleWrap, interface IConsole

Reviewer: I think there are three back ticks missing here to close this code block. I cannot add these as a suggestion because GitHub does not display these backticks.
